api/v1/verticadb_webhook.go

```diff
@@ -106,6 +106,7 @@ func (v *VerticaDB) ValidateCreate() error {
 	verticadblog.Info("validate create", "name", v.Name, "GroupVersion", GroupVersion)
 
 	allErrs := v.validateVerticaDBSpec()
+	allErrs = v.checkNewSBoxOrSClusterShutdownUnset(allErrs)
```

Comment:
Move this to `validateVerticaDBSpec()` and remove it from `validateImmutableFields()`.